src/test/java/divaeva/hw/api/store/StoreTestBase.java

```diff
@@ -0,0 +1,18 @@
+package divaeva.hw.api.store;
+
+import dto.StoreDTO;
+
+public class StoreTestBase {
+    protected final String BASE_SERVER_URL_STORE = "https://petstore.swagger.io/v2/store/order/";
+    protected final String BASE_SERVER_URL_STORE_INVENTORY = "https://petstore.swagger.io/v2/store/inventory";
+
+    protected StoreDTO createDefaultStore(int id) {
```

Comment:
Why not to use constructor int "StoreDTO" class for creating default store?

Reply:
fixed